pkg/exchange/binance/exchange.go

```diff
@@ -35,11 +35,22 @@ const DefaultDepthLimit = 5000
 const BinanceUSBaseURL = "https://api.binance.us"
 const BinanceTestBaseURL = "https://testnet.binance.vision"
 const BinanceUSWebSocketURL = "wss://stream.binance.us:9443"
+
 const WebSocketURL = "wss://stream.binance.com:9443"
-const WebSocketTestURL = "wss://testnet.binance.vision"
-const FutureTestBaseURL = "https://testnet.binancefuture.com"
 const FuturesWebSocketURL = "wss://fstream.binance.com"
-const FuturesWebSocketTestURL = "wss://stream.binancefuture.com"
+const TestNetFuturesWebSocketURL = "wss://stream.binancefuture.com"
```

Comment:
This one does not look like testnet url.
Is it on purpose or a typo?

Reply:
It's just a rename from the above line<